main.lua

```diff
@@ -453,6 +462,16 @@ function restore_camera()
 	love.graphics.translate(-pico8.camera_x, -pico8.camera_y)
 end
 
+local function menu_print(str,x,y)
+	love.graphics.setShader(pico8.sysfont_shader)
+	str=tostring(str):gsub("[%z\1-\9\11-\31\154-\255]", ""):gsub("[\128-\153]", "\194%1").."\n"
+	local size=0
+	for line in str:gmatch("(.-)\n") do
+		love.graphics.print(line, x, y+size)
+		size=size+6
+	end
+end
+
```

Comment:
Since we can't use the api.print because we need the new shader (and so that we don't mess up any part of pico8's drawstate), I just added this which is a simplified print with the correct shader